build/opt/pdo/etc/template/site.psh

```diff
@@ -26,11 +26,11 @@ fi
 ## will be used by the local client to create & interact with
 ## contracts.
 ## -----------------------------------------------------------------
-set -s eservice1 -v http://localhost:7101
-set -s eservice2 -v http://localhost:7102
-set -s eservice3 -v http://localhost:7103
-set -s eservice4 -v http://localhost:7104
-set -s eservice5 -v http://localhost:7105
+set -s eservice1 -v http://SERVICE_HOST:7101
```

Comment:
Where is `SERVICE_HOST` coming from? We don't seem to use this anywhere else.

Reply:
this is supposed to be modified in the Makefile when it is installed. i'm sure there is a better way to do this. site.psh is really a client thing. so PDO_HOSTNAME (the default value) is not quite what we want. as is, it should create the environment necessary for running tests. and it can be (should be) modified by a "real" client with the list of interesting services.